HW2/P5/P5.txt

```diff
@@ -0,0 +1,86 @@
+1) Serial run yields performance of between 5 to 6 simulations per second, while parallel run yields 7-8 simulations per second. Therefore performance is better for 4 thread case, but not four times.
```

Comment:
Exact same speed-ups I got. My guess for explanation is not 4x better because of wait times + overhead.